### PR TITLE
Ruff format tests

### PR DESCRIPTION
**Pull request by niccokunzmann**
This is a contribution to #672 and formats the test files.

Some errors are still in the tests files. I started with formatting these as they likely do not cause merge conflicts for future contributions.

----
📚 Documentation preview 📚: https://icalendar--719.org.readthedocs.build/

### CODE REVIEW

**Comment by niccokunzmann** [comment]
@jacadzaca, if you have time, please review it.
@stevepiercy, this is for #672 in which we had a discussion about code styling.

File: CHANGES.rst [hunk 1]
```diff
@@ -6,7 +6,7 @@ Changelog
 
 Minor changes:
 
-- ...
+- Ruff format test code, see `Issue 672 <https://github.com/collective/icalendar/issues/672>`_
```
Comment (stevepiercy) [inline CHANGES.rst]:
```suggestion
- Format test code with Ruff. See `Issue 672 <https://github.com/collective/icalendar/issues/672>`_.
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Proper grammar on change log. If tests pass, LGTM!

**Comment by niccokunzmann** [comment]
@stevepiercy Thanks for the review! When the tests pass, I will merge it.

**Pull request merged**